pubspec.yaml

```diff
@@ -1,13 +1,16 @@
 name: pool
-version: 1.3.5
+version: 1.3.6-dev
```

Comment:
We shouldn't need to bump this since there are no changes under lib?

Reply:
I always bump the version so it's clear we've moved beyond the last release.